src/components/shared/MembersList/Actions/MemberActionsPopover.tsx

```diff
@@ -0,0 +1,125 @@
+// import { useDialog } from '~shared/Dialog';
+// import { BanUserDialog } from '~shared/Comment';
+import ExternalLink from '~shared/ExternalLink';
+// import ManageWhitelistDialog from '~dashboard/Dialogs/ManageWhitelistDialog';
+
+import styles from './MemberActionsPopover.css';
+
+const displayName = 'MembersList.Actions.MemberActionsPopover';
+
+const MSG = defineMessages({
+  addToAddressBook: {
+    id: `${displayName}.addToAddressBook`,
+    defaultMessage: `Add to address book`,
+  },
+  banUser: {
+    id: `${displayName}.banUser`,
+    defaultMessage: `{unban, select,
+      true {Unban}
+      other {Ban}
+    } user`,
+  },
+  viewOnBlockscout: {
+    id: `${displayName}.viewOnBlockscout`,
+    defaultMessage: 'View on Blockscout',
+  },
+});
+
+interface Props {
+  closePopover: () => void;
+  userAddress: string;
+  isWhitelisted: boolean;
+  isBanned: boolean;
+  canAdministerComments?: boolean;
+}
+
+const MemberActionsPopover = ({
+  closePopover,
+  canAdministerComments,
+  userAddress,
+  isWhitelisted,
+  isBanned,
+}: Props) => {
+  // const openBanUserDialog = useDialog(BanUserDialog);
+
+  // const handleBanUser = useCallback(
+  //   () =>
+  //     openBanUserDialog({
+  //       colonyAddress: colony.colonyAddress,
+  //       isBanning: !isBanned,
+  //       addressToBan: userAddress,
+  //     }),
+  //   [openBanUserDialog, colony, userAddress, isBanned],
+  // );
+  const handleBanUser = () => console.log('Ban user dialog');
+
+  // const openManageWhitelistDialog = useDialog(ManageWhitelistDialog);
+  // const handleManageWhitelist = useCallback(
+  //   () => openManageWhitelistDialog({ userAddress, colony }),
+  //   [openManageWhitelistDialog, userAddress, colony],
+  // );
+  const handleManageWhitelist = () => console.log('Whitelist dialog');
+
+  const renderUserActions = () => (
+    <DropdownMenuItem>
+      <Button appearance={{ theme: 'no-style' }}>
+        <ExternalLink
+          href={getBlockscoutUserURL(userAddress)}
+          className={styles.actionButton}
+          text={MSG.viewOnBlockscout}
+        />
+      </Button>
+    </DropdownMenuItem>
+  );
+
+  const renderModeratorOptions = () => {
+    return (
+      <>
+        {!isWhitelisted && (
+          <DropdownMenuItem>
+            <Button
+              appearance={{ theme: 'no-style' }}
+              onClick={() => handleManageWhitelist()}
+            >
+              <div className={styles.actionButton}>
+                <FormattedMessage {...MSG.addToAddressBook} />
+              </div>
+            </Button>
+          </DropdownMenuItem>
+        )}
+        <DropdownMenuItem>
+          <Button
+            appearance={{ theme: 'no-style' }}
+            onClick={() => handleBanUser()}
+          >
+            <div className={styles.actionButton}>
+              <FormattedMessage {...MSG.banUser} values={{ unban: isBanned }} />
+            </div>
+          </Button>
+        </DropdownMenuItem>
+      </>
+    );
+  };
```

Comment:
We need to decide what to do with the Whitelist functionality, but since we're not prioritizing comments for the initial launch, all the comment ban/remove logic can go for now

Reply:
Removed whitelist, ban/remove logic